Week2/homework/maartjes-work.js

```diff
@@ -42,20 +42,72 @@ const tuesday = [
   },
 ];
 
-const maartjesTasks = monday.concat(tuesday);
+// I changed maartjesTasks variable into let in this exercise because I wanted to make changes in the original array.
+// I know that is not good practice but since at the end maartjesTasks and maartjesHourlyRate is used, I thought I should go from that.
+
+let maartjesTasks = monday.concat(tuesday);
```

Comment:
The other way not to modify `maartjesTasks` is to put your code (line 77-90)  into the `computeEarnings()` function. I would suggest this as it keeps all the earnings calculations code into one function.

Reply:
I applied this change now and it feels and looks better thank you!